src/backends/ffmpeg.moon

```diff
@@ -0,0 +1,108 @@
+class FfmpegBackend extends Backend
+	new: =>
+		@name = "ffmpeg"
+
+	-- Turn `MpvFilter`s into command line options.
+	solveFilters: (filters) =>
+		solved = {}
+		for filter in *filters
+			if not filter.lavfiCompat
+				continue
+			str = filter.name .. "="
+			for k, v in pairs filter.params
+				str ..= "#{v}:"
```

Comment:
Shouldn't just ignore `k` here.

Reply:
Again, I'm an amateur and while I use ffmpeg a lot it's for basic things. I'm not really familiar with using complex filters. I was under the impression, however, that the `-vf` option doesn't use keys, at least not in the way mpv does, and just uses `name=arg0:arg1:arg2...,name2=...` [That's totally wrong though looking at it](https://ffmpeg.org/ffmpeg-filters.html#Filtergraph-syntax-1), so I'll fix this.